qutebrowser/completion/completionwidget.py

```diff
@@ -105,9 +107,10 @@ class CompletionView(QTreeView):
     update_geometry = pyqtSignal()
     selection_changed = pyqtSignal(str)
 
-    def __init__(self, win_id, parent=None):
+    def __init__(self, win_id: int,
+                 parent: typing.Optional[QWidget] = None) -> None:
```

Comment:
We don't turn on `no_implicit_optional` in mypy, so you could just do `parent: QWidget = None` which is a bit shorter (and the `= None` makes it immediately clear that it can be `None` as well).

Reply:
these type hints aren't actually here because I wanted to add them, but because otherwise pylint complains about unused imports (and flake8 complains about missing imports). I could silence the warnings instead if you want.